configuration/rules-blockly.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: documentation
+title: Rules Blockly
+---
+
+# Blockly Reference
+
+One of the core feature that OpenHAB provides is writing rules to allow specific behaviour with the home automation system. The usual way of developing rules is by coding them like described in the [Textual Rules]({{base}}/configuration/rules-dsl.html). However, this art of programming may become intimidating early on and shy away away people with few or almost no experience in programming. Therefore openHAB also provides a graphical way of writing rules which allows to put together rules in a rather visual way (even though some programming background may still help).
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.